Bottom: 19a2159bff1a1d081b09f0850cc3c975e3bc91b6
Top:    ae4a3a7ff72e000eeab9d7762a7417c6622ef1e7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 04:49:57 +0100

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
